plugins/UsersManager/angularjs/personal-settings/personal-settings.controller.js

```diff
@@ -34,8 +34,26 @@
             });
         }
 
+        this.regenerateTokenAuth = function () {
+
+            self.loading = true;
+
+            piwikApi.withTokenInUrl();
+            piwikApi.post({
+                module: 'UsersManager', action: 'regenerateTokenAuth', format: 'json'
```

Comment:
:white_check_mark: (marking typo for myself before updating the PR)

Was already done in `manage_users.controller.js`. So the controller was not needed in the first place.